core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -348,10 +354,10 @@ object AppType {
    */
   def appTypeToFormattedByType(appType: AppType): FormattedBy =
     appType match {
-      case Galaxy                     => FormattedBy.Galaxy
-      case Custom                     => FormattedBy.Custom
-      case RStudio                    => FormattedBy.RStudio
-      case Cromwell | Wds | HailBatch => FormattedBy.Cromwell
+      case Galaxy                                                        => FormattedBy.Galaxy
+      case Custom                                                        => FormattedBy.Custom
+      case RStudio                                                       => FormattedBy.RStudio
+      case Cromwell | Wds | HailBatch | WorkflowsApp | CromwellRunnerApp => FormattedBy.Cromwell
```

Comment:
I know you're just following the existing pattern here, but... weird pattern, right?

Reply:
Hmm, yeah.
1. I believe this is also entirely unnecessary for these apps which won't really do anything with their persistent disks
2. But might also cause problems if we ever try to run both apps in the same workspace, will they be fighting over disks they don't need 🤔

Reply:
I guess obviously it's not a problem since Cromwell, WDS and HailBatch are already sharing this, but I don't think there's any reason not to have more cases here

Reply:
this can definitely be simplified...we pretty much only care if the app is galaxy, and don't wanna reuse a disk used by galaxy for other apps...all other cases are probably ok